mrp_bom_state/i18n/es.po

```diff
@@ -29,7 +29,7 @@ msgstr "Activa"
 #. module: mrp_bom_state
 #: field:mrp.config.settings,group_mrp_bom_state:0
 msgid "Allow re-edit the BoM list"
-msgstr "Permitir la re-editar la lista de lista de materiales"
+msgstr "Permitir re-editar la lista de lista de materiales"
```

Comment:
Ya que lo cambias, ponlo mejor como "Permitir re-editar las listas de materiales", y el texto en inglés sería "Allow to re-edit BoMs"